apollo-federation/src/query_graph/condition_resolver.rs

```diff
@@ -23,9 +23,13 @@ pub(crate) struct ContextMapEntry {
     pub(crate) levels_in_query_path: usize,
     pub(crate) path_tree: Option<Arc<OpPathTree>>,
     pub(crate) selection_set: SelectionSet,
-    pub(crate) param_name: Name,
```

Comment:
Refactors:
- Some renames here to match with `ContextCondition` and `QueryGraph`.